scipy/spatial/transform/rotation.py

```diff
@@ -1864,9 +1866,14 @@ def match_vectors(cls, a, b, weights=None, normalized=False):
                                     weights.shape[0], b.shape[0]))
         weights = weights / np.sum(weights)
 
-        if not normalized:
-            a = a / scipy.linalg.norm(a, axis=1)[:, None]
-            b = b / scipy.linalg.norm(b, axis=1)[:, None]
+        if normalized is not None:
```

Comment:
This whole block is broken, need to make a decision about the change as I explained above.